sdk/inc/azure/platform/az_mqtt_mosquitto.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ *
+ * @brief Defines MQTT constructs for Mosquitto.
+ *
+ * @note You MUST NOT use any symbols (macros, functions, structures, enums, etc.)
+ * prefixed with an underscore ('_') directly in your application code. These symbols
+ * are part of Azure SDK's internal implementation; we do not document these symbols
+ * and they are subject to change in future versions of the SDK which would break your code.
+ */
+
+#ifndef _az_MQTT_MOSQUITTO_H
+#define _az_MQTT_MOSQUITTO_H
+
+#include "mosquitto.h"
+#include <azure/core/internal/az_mqtt_internal.h>
+
+#include <azure/core/_az_cfg_prefix.h>
+
+typedef struct
+{
+  az_mqtt_options_common platform_options;
+  az_span openssl_engine;
+  // HFSM_TODO: add initial mosquitto_handle here.
```

Comment:
Could you please implement this part?
If the app requires an initial handler, it should be passed in options not in the az_mqtt struct.

Reply:
Will require a small change in `az_mqtt_init`.